### ISSUE TITLE
GPS Points KML file failing to download

### ISSUE THREAD

**Issue by aprsandrew** [aprsworld/craneWeb#34]
On units with large number of GPS data points (not sure of exact number, but possibly in neighborhood of 60,000 data points), the points KML file (gpsKML.php) will fail to download.  Although with that many data points, it is probably too large of a KML file to reasonably work with anyway.  

Quick fix may be to simply limit the number of data points that are downloaded (although it should be the x number of *newest* data points).  Note this is also somewhat related to issue #23 

Would be nice to combine fix for this with an enhancement:
Should have a dialog box with the following options to specify for downloading a limited set of data:
start date / end date
min/max latitude and longitude
lines or points

In other words, the ability to limit to a specific time period or specific region.  This would make it much easier to download data for only a specific project area or time period.

**Comment by aprsandrew** [comment]
Also of note, downloads should probably be from the individual gpsUp_station_A1234 tables versus the combined gpsUp table to reduce server load / wait time.
